packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx

```diff
@@ -29,68 +32,87 @@ const SlackSettingsPanel: FC = () => {
     const installUrl = `/api/v1/slack/install/`;
 
     if (isLoading) {
-        return <Spinner />;
+        return <Loader />;
     }
 
     const isValidSlack = data?.slackTeamName !== undefined && !isError;
     return (
-        <SlackSettingsWrapper>
-            <SlackIcon src={slackSvg} />
-            <AppearancePanelWrapper>
-                <Title> Slack integration</Title>
+        <SettingsGridCard>
+            <div>
```

Comment:
I was thinking about this the other day, do we want to replace `div`s with `Box` component or we don't care much about that in this case? 🤔

Reply:
if there's nothing mantine specific then its just a unnecessary complexity